lib/methods/import.js

```diff
@@ -792,7 +861,7 @@ module.exports = self => {
 
       const { docs, attachmentsInfo = [] } = await format.input(exportPath);
 
-      const filterDocs = docs.filter(doc => docIds.includes(doc.aposDocId));
+      let filterDocs = docs.filter(doc => docIds.includes(doc.aposDocId));
```

Comment:
Could avoid reassigning:
```javascript
      const filterDocs = importDraftsOnly
        ? self.getPublishedDocsAsDraft(
          docs.filter(doc => docIds.includes(doc.aposDocId))
        )
        : docs.filter(doc => docIds.includes(doc.aposDocId));
```

Reply:
done, and filtering now happens before locale re-writing